docs/source/reference/rest_api.rst

```diff
@@ -184,7 +97,7 @@ Nodes
 
     Description:
 
-        returns the list of *Node* objects. Every node object contains value of attribute called ``pbc1`` if present otherwise null.
+        returns the list of |Node| objects. Every node object contains value of attribute called ``pbc1`` if present otherwise null.
```

Comment:
```suggestion
        returns the list of |Node| objects.
        Every node object contains value of attribute called ``pbc1`` if present otherwise null.
```